lib/internal/Magento/Framework/Translate.php

```diff
@@ -460,6 +460,18 @@ public function getTheme()
         return self::CONFIG_THEME_KEY . $theme['theme_title'];
     }
 
+    /**
+     * Set theme
+     *
+     * @param string $theme
+     * @return \Magento\Framework\TranslateInterface
+     */
+    public function setTheme($theme)
```

Comment:
It's a bad practice to introduce public methods for using them in tests. Please remove it